x-pack/plugins/security_solution/server/endpoint/migrations/ensure_indices_exists_for_policies.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createPolicyDataStreamsIfNeeded } from '../../fleet_integration/handlers/create_policy_datastreams';
+import type { EndpointAppContextService } from '../endpoint_app_context_services';
+
+export const ensureIndicesExistsForPolicies = async (
+  endpointServices: EndpointAppContextService
+): Promise<void> => {
+  const logger = endpointServices.createLogger('startupPolicyIndicesChecker');
+
+  const fleetServices = endpointServices.getInternalFleetServices();
+  const soClient = fleetServices.savedObjects.createInternalUnscopedSoClient();
+  const endpointPoliciesIds = await fleetServices.packagePolicy.listIds(soClient, {
+    kuery: fleetServices.endpointPolicyKuery,
+    perPage: 10000,
```

Comment:
What is the maximum number of items returned here? Might be a good idea to paginate if 100+.

Kibana runs on pretty low resources in some environments and might be doing other things at the same time.

Reply:
Thanks for 👀 
The likelihood - especially for serverless - is that the returned count of items will be low. maybe a handful of policies.